packages/integration-tests/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "integration-tests",
+  "version": "0.0.0",
+  "title": "Integration tests",
+  "description": "The test solution for transaction processing`",
```

Comment:
Is it just for transaction processing? Should it be `The test solution for Block Explorer API and UI`?

Reply:
updated